controllers/searchoperator_controller.go

```diff
@@ -67,6 +68,7 @@ var (
 	storageClass         = ""
 	storageSize          = "10Gi"
 	namespace            = os.Getenv("WATCH_NAMESPACE")
+	setupPod, _          = strconv.ParseBool(os.Getenv("UPSTREAM"))
```

Comment:
Maybe name the env variable something like `DEPLOY_REDISGRAPH` or `USE_EXISTING_REDISGRAPH`